### ISSUE TITLE
mozilla-version-comparator module is licensed under GPL 3.0

### ISSUE THREAD

**Issue by ngyikp** [mozilla-jetpack/jpm#612]
https://github.com/linagora/mozilla-version-comparator

Could this module be replaced with another one that has a more permissive license?

In fact, it seems that this module isn't being used anymore, a quick [GitHub code search](https://github.com/mozilla-jetpack/jpm/search?utf8=%E2%9C%93&q=mozilla-version-comparator&type=) shows no other usage, [this commit](https://github.com/mozilla-jetpack/jpm/commit/a538c9d5c3734eb93aa587d98249a3eb501ae0d7#diff-50e3aa130a4f97a42ee2cf111c7b1d9d) from last year removed the usage of it.

**Comment by kumar303** [comment]
Yep, looks like `mozilla-version-comparator` can be removed from `package.json` entirely
